pkg/server/logger.go

```diff
@@ -81,9 +90,21 @@ func makeDefaultLogger(lvl logging.Level, fmt logging.Format) (log.Logger, error
 func (l *Logger) Log(kvps ...interface{}) error {
 	l.mut.RLock()
 	defer l.mut.RUnlock()
+	err := l.HookLogger.Log(kvps...)
+	if err != nil {
+		return err
+	}
 	return l.l.Log(kvps...)
 }
 
+// Log implements log.Logger.
+func (hl *HookedLogger) Log(kvps ...interface{}) error {
+	if hl.Enabled {
+		return hl.Logger.Log(kvps...)
+	}
+	return nil
```

Comment:
This needs a mutex to prevent concurrent read/writes in the middle of a support bundle generation while subsystems are actively logging.